tests/integration/test_layout.py

```diff
@@ -1,68 +0,0 @@
-# -*- coding: utf-8 -*-
-#
-# Copyright (C) 2015 McKinsey Academy
-#
-# Authors:
-#          Jonathan Piacenti <jonathan@opencraft.com>
-#
-# This software's license gives you freedom; you can copy, convey,
-# propagate, redistribute and/or modify this program under the terms of
-# the GNU Affero General Public License (AGPL) as published by the Free
-# Software Foundation (FSF), either version 3 of the License, or (at your
-# option) any later version of the AGPL published by the FSF.
-#
-# This program is distributed in the hope that it will be useful, but
-# WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
-# General Public License for more details.
-#
-# You should have received a copy of the GNU Affero General Public License
-# along with this program in a file in the toplevel directory called
-# "AGPLv3".  If not, see <http://www.gnu.org/licenses/>.
-#
-
-"""
-Test to make sure the layout for results is sane when taking images into
-account.
-"""
-from tests.integration.base_test import PollBaseTest
-
-
-class TestLayout(PollBaseTest):
```

Comment:
@Kelketek Is there a way to (partially) preserve layout tests? Or to replace them with a test that ensures that multi-line answers don't break the layout?

Completely removing these tests seems a bit radical to me (but then again I don't know this code base very well, so I might be missing something).